docs/docs/guides/web3_plugin_guide/plugin_users.md

```diff
@@ -0,0 +1,177 @@
+---
+sidebar_position: 1
```

Comment:
```suggestion
sidebar_position: 2
```

Reply:
[This](https://github.com/web3/web3.js/pull/5558/commits/403ad79a9fb761287c6102c427a4cfd57cc8da44) commit updates the plugin author's guide to position `0`